File: 013-spring-boot-flink-java/src/main/java/com/learn/flink/wc/NCWordCountApp.java
package com.learn.flink.wc;
import org.apache.flink.streaming.api.datastream.DataStreamSource;
import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;

public class NCWordCountApp {

    public static void main(String[] args) throws Exception {
        // 1.获取运行环境
        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
        DataStreamSource<String> ds = env.socketTextStream("47.94.80.7", 9999);
        ds.print();
        env.execute("dddd");
    }
}

File: 013-spring-boot-flink-scala/src/main/scala/com/learn/flink/NCWordCountApp.scala
package com.learn.flink

import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment
import org.apache.flink.api.scala._

object NCWordCountApp {

  def main(args: Array[String]): Unit = {
    val env = StreamExecutionEnvironment.getExecutionEnvironment;
    val ds = env.socketTextStream("47.94.80.7", 9999);
    ds.print().setParallelism(1)
    env.execute("NCWordCountAppExecute")
  }
}
